apps/server/src/modules/task-card/uc/task-card.uc.spec.ts

```diff
@@ -427,5 +439,18 @@ describe('TaskCardUc', () => {
 			expect((result.card.cardElements.getItems()[0] as RichTextCardElement).value).toEqual(richText[0]);
 			expect((result.card.cardElements.getItems()[1] as RichTextCardElement).value).toEqual(richText[1]);
 		});
+		it('should throw if course has no end date', async () => {
+			course = courseFactory.buildWithId({ untilDate: undefined });
+			courseRepo.findById.mockResolvedValue(course);
+			const TaskCardCreateParams = {
```

Comment:
`TaskCardCreateParams` should be named `taskCardUpdateParams` according to the situation (also take the camel case notation in mind here as well as above)